### ISSUE TITLE
[PeoplePicker] - NVDA Screen reader doesn't read when an email has been removed (FluentUI v8)

### ISSUE THREAD

**Issue by NathanC15** [microsoft/fluentui#21376]
### Environment Information

- **Package version(s)**: 8.41.2
- **Browser and OS versions**: Microsoft Edge

### Describe the issue:

When using NVDA screen reader and you use keyboard navigation to go to the "remove" button to remove an email from the people picker. If you listen to the screen reader, it never reads that the email has been removed. What's happening is that the people picker is taking focus of the input box to quickly after you click the "remove" button which results the screen reader to read "People picker combo box collapsed has auto complete editable blank" instead of "removed (email)". I did notice that the screen reader does notice the "removed (email)" but doesn't have enough time to read it due to it taking focus of the input box.

#### Actual behavior:

NVDA Screen reader doesn't read to the user "removed (email)" instead reads "People picker combo box collapsed has auto complete editable blank"

#### Expected behavior:

NVDA Screen reader to say to the user "removed (email)"

### Documentation describing expected behavior

https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

**Comment by gouttierre** [comment]
@NathanC15 - Thanks for filing this issue with us. To set expectations, the developers will review this issue once capacity allows as we’re currently focused on larger coherence work items.

@TristanWatanabe - Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by TristanWatanabe** [comment]
Can confirm that this is an issue. Screen reader does not announce that a picker item has been removed when using the keyboard and navigating to the remove button. It works fine when using the mouse and clicking on the remove button though.

**Comment by smhigley** [comment]
I can't reproduce this with NVDA in Edge, Chrome, or Firefox -- the live region works in each of those as shown in this video:

https://user-images.githubusercontent.com/3819570/168657774-4dfda1af-a5df-4ead-9ac1-b65449a5b654.mp4

@NathanC15 is this still reproducing for you? And if so, could you specify NVDA and Edge versions?
